src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -220,8 +220,16 @@ public void testInputValidation() {
         feedbackEditPage.clickAssignWeightsCheckboxForNewQuestion();
         feedbackEditPage.fillRubricWeightBoxForNewQuestion("", 3);
         feedbackEditPage.clickAddQuestionButton();
-
-        feedbackEditPage.waitForTextsForAllStatusMessagesToUserEquals(Const.FeedbackQuestion.RUBRIC_ERROR_INVALID_WEIGHT);
+        assertTrue(feedbackEditPage.isRubricWeightBoxFocused(questionNum, 3));
+        feedbackEditPage.fillRubricWeightBoxForNewQuestion("0", 3);
+        feedbackEditPage.clickAddRubricColLink(questionNum);
+        feedbackEditPage.fillRubricWeightBoxForNewQuestion("", 4);
+        feedbackEditPage.clickAddQuestionButton();
+        assertTrue(feedbackEditPage.isRubricWeightBoxFocused(questionNum, 4));
+        feedbackEditPage.clickRemoveRubricColLinkAndConfirm(questionNum, 4);
+        feedbackEditPage.clickAssignWeightsCheckboxForNewQuestion();
```

Comment:
Test that if you unchecked the weight checkbox and you are able to submit even though the weight is empty?

Reply:
Should I test this in the same `______TS` or should I create a separate test? I guess having a separate test will be better don't you think?

Reply:
Added the test in the same test case and separated the lines with comments so that the test is easier to read.